build.cmd

```diff
@@ -158,6 +160,8 @@ if [!processedArgs!]==[] (
 
 :ArgsDone
 
+if %__PgoOptimize%=0 set __RestoreOptData=0
```

Comment:
Wrong syntax--you need double-equals. To keep with script conventions, I recommend either
```batch
if "%__PgoOptimize%" == "0" set __RestoreOptData=0
```
or
```batch
if [%__PgoOptimize%] == [0] set __RestoreOptData=0
```